tests/integration/test_auth_ldap.py

```diff
@@ -34,9 +34,13 @@ def setUp(self):
 
             'LDAP_GROUP_BASEDN': 'ou=people,dc=planetexpress,dc=com',
             'LDAP_GROUP_FILTER': '(&(member={userdn})(objectClass=group))',
-            'LDAP_GROUP_NAME_ATTR': 'cn',  # memberOf or cn
+            # 'LDAP_GROUP_NAME_ATTR': 'cn',  # memberOf or cn or None
 
-            'LDAP_DEFAULT_DOMAIN': 'planetexpress.com'
+            'LDAP_DEFAULT_DOMAIN': 'planetexpress.com',
+            'ALLOWED_LDAP_GROUPS': [
+                'cn=admin_staff,ou=people,dc=planetexpress,dc=com',
+                'cn=ship_crew,ou=people,dc=planetexpress,dc=com'
```

Comment:
Users need to belong to groups with full dn match, not just cn attribute, unless specified eg. using "memberOf"